packages/flutter/lib/src/rendering/flex.dart

```diff
@@ -776,8 +776,8 @@ class RenderFlex extends RenderBox with ContainerRenderObjectMixin<RenderBox, Fl
   Rect describeApproximatePaintClip(RenderObject child) => _overflow > 0.0 ? Offset.zero & size : null;
 
   @override
-  String toString() {
-    String header = super.toString();
+  String toShortDescription() {
```

Comment:
I've switched the toString for RenderObject back. However, it does seem like a lot of error messages were already using toStringShort or toStringDeep rather than toString for RenderObject.
However, i don't know a reliable way in IntelliJ to find all usages for RenderObject.toString so I can't say for sure.